docs/cpu.md

```diff
@@ -0,0 +1,60 @@
+# CPU example usage
+
+This document describes an example use of the [cpu](https://github.com/u-root/cpu)
+command.
+
+## Introduction
+
+The `cpu` command allows you to log in to a remote system (in this case RTE) and
+see some of the files from your local system. This allows you to execute
+programs installed on your local system in the remote environment. Another use
+case is executing `minicom` on RTE and storing the logs on your host PC without
+having to `scp` them and without worrying about RTE running out
+of space. This use case is demonstrated in this document.
+
+## Prerequisites
+
+* RTE with system version `0.7.4-rc5` or later connected to your local network
+* Some bootable platform connected to the RTE via serial
+* The `cpu` command installed on your host PC
+
+## Using cpu
+
+This section describes how to use the `cpu` command to execute `minicom` on RTE
```

Comment:
Let's point here to some advantages, e.g. we do not need to worry about ending storage space on RTE.

Reply:
I already mentioned this in the introduction. Should I repeat it?

Reply:
i mentioned it again e2e288c2cfe632e4be4be87116ed66b448439d17